### PR TITLE
Add support for yamllint package

### PR DESCRIPTION
**Pull request by DmytroLitvinov**
Hi @sobolevn. This MR fix some of errors from default settings of `yamllint` package at all yaml files.
Also add some notes to docs.

Please verify it and double-check `Pipenv.lock` file, because `pipenv lock` generated something different that was before.

### CODE REVIEW

**Comment by DmytroLitvinov** [comment]
This MR for https://github.com/wemake-services/wemake-django-template/issues/425

File: {{cookiecutter.project_name}}/.yamllint [hunk 1]
```diff
@@ -0,0 +1 @@
+extends: default
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/.yamllint]: Is there any way to get rid of this file? Like default configuration or `setup.cfg` or something?

File: {{cookiecutter.project_name}}/docker/docker-compose.prod.yml [hunk 2]
```diff
@@ -27,7 +27,11 @@ services:
 
   web:
     <<: &web
-      image: "registry.gitlab.com/{{ cookiecutter.organization }}/{{ cookiecutter.project_name }}:latest"
+      image:
+        "registry.gitlab.com/\
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docker/docker-compose.prod.yml]: Is this a linting recommendation? I don't like it, really.
I guess I have to go through the config values to find out the perfect fit.

File: {{cookiecutter.project_name}}/docs/_pages/template/linters.rst [hunk 3]
```diff
@@ -79,4 +79,5 @@ yamllint
 ~~~~~~~~
 
 This linter is used to lint your `yaml` files.
+Just run ``yamllint .``.
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docs/_pages/template/linters.rst]: It is a good practice (I try really hard to follow it) not to start your sentences with "just".

File: {{cookiecutter.project_name}}/.yamllint (hunk 1 above)
Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/.yamllint]: I think we can remove it.

Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/.yamllint]: From docs:

If -c is not provided, yamllint will look for a configuration file in the following locations (by order of preference):

.yamllint in the current working directory
$XDG_CONFIG_HOME/yamllint/config
~/.config/yamllint/config



So remove it from PR?
